tests/manage/mcg/test_verify_noobaa_status.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from ocs_ci.ocs import defaults
+from ocs_ci.utility.utils import run_async
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1, noobaa_cli_required
+)
+from ocs_ci.framework.testlib import polarion_id, bugzilla
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@noobaa_cli_required
+@polarion_id('OCS-2084')
+@bugzilla('1799077')
+def test_verify_noobaa_status():
+    """
+    Verify noobaa status output is clean without any errors
+    """
+    contents_count = [
+        'CustomResourceDefinition-5', 'Namespace-1', 'ServiceAccount-1',
```

Comment:
I hope the numbers dont change across newer builds.

Reply:
@jilju could you please save these in defaults.py?

Reply:
Done